src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+### Light theme
+
+Gray 10 components are generally paired with White backgrounds and White components with Gray 10 backgrounds. Some components, such as buttons and toggles, are common across both backgrounds. 
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 10 Dropdown on white background.">
+        <img src="images/Light_theme_01.png" alt="Gray 10 Dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="White Dropdown on a Gray 10 background.">
+        <img src="images/Light_theme_revised.png" alt="White Dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="White Overflow Menu on Gray 10 card.">
+        <img src="images/Light_theme_03_new.png" alt="White Overflow Menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Avoid use of midtones.">
+        <img src="images/Light_theme_04.png" alt="Avoid use of midtones.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Light theme sample](images/Light_theme_sample_revised.gif)
+
+
+_Components with common colors across both light UI backgrounds_
+
+
+### Dark theme
+
+Gray 90 components are generally paired with Gray 100 backgrounds and Gray 80 components with Gray 90 backgrounds. Some components, such as buttons and toggles, are common across all three dark UI background.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 90 Dropdown on Gray 100 background.">
+        <img src="images/Dark_theme_01_new.png" alt="Gray 10 Dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="Gray 80 Dropdown on Gray 90 background.">
+        <img src="images/Dark_theme_02_rev.png" alt="White Dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 70 is sometimes used when compound components are being applied such as an overflow menu.">
+        <img src="images/Dark_theme_03_new.png" alt="White Overflow Menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Do not apply components that are darker than the background unless using high contrast mode.">
+        <img src="images/Dark_theme_01_new.png" alt="Do not apply components that are darker than the background unless using high contrast mode.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Dark theme sample](images/Dark_theme_sample_revised.gif)
+
+_Components with common colors across both dark UI backgrounds_
+
+### High contrast
+
+In some cases, it is helpful to apply light components to dark backgrounds or light components to dark backgrounds. This technique is useful to focus attention or create visual tension when necessary.
```

Comment:
In some cases, it is helpful to apply light components to dark backgrounds or ~~light~~ **dark** components to ~~dark~~ **light** backgrounds.